ts/Dashboards/Layout/Cell.ts

```diff
@@ -501,27 +505,62 @@ class Cell extends GUIElement {
 }
 
 namespace Cell {
-
+    /**
+     * @internal
+     **/
     export interface CellResponsiveOptions {
         width: string;
         // visible: boolean;
     }
 
+    /**
+     * @internal
+     **/
     export interface JSON extends Serializable.JSON<'Dashboard.Layout.Cell'> {
         options: OptionsJSON;
     }
 
+    /**
+     * Options for each cell.
+     **/
     export interface Options {
+        /**
+         * Unique cell id.
+         **/
         id: string;
-        width?: string; // eg 50%, 1/2
+        /**
+         * With of the cell. Can be a percentage value or a fraction.
```

Comment:
Typo: `width`

Reply:
I would add examples here, in the description. Simple `For example '50%' or '1/3'` would be enough, what do you think?